test/bddtests/fixtures/config/fabric/sidetree-protocol-v0_1_2.json

```diff
@@ -0,0 +1,14 @@
+{
+  "genesisTime": 1000,
+  "hashAlgorithmInMultihashCode": 18,
+  "hashAlgorithm": 5,
+  "maxOperationSize": 300000,
+  "maxOperationCount": 50,
+  "compressionAlgorithm": "GZIP",
+  "maxAnchorFileSize": 2000000,
+  "maxMapFileSize": 2000000,
+  "maxChunkFileSize": 20000000,
+  "signatureAlgorithms": ["EdDSA", "ES256", "ES256K"],
+  "keyAlgorithms": ["Ed25519", "P-256", "secp256k1"],
+  "enableReplacePatch": true
```

Comment:
I think we should revisit the one-off patch flag. Perhaps it should be generic: https://github.com/trustbloc/sidetree-core-go/issues/423